fiftyone/plugins/core.py

```diff
@@ -488,9 +492,18 @@ def _parse_plugin_metadata(metadata_path):
     return PluginPackage(plugin_name, plugin_path)
 
 
-def _list_plugins(enabled=None):
+def _list_plugins(enabled=None, include_builtin=False):
     plugins = []
-    for metadata_path in _iter_plugin_metadata_files():
+    external_plugin_paths = list(_iter_plugin_metadata_files())
+    builtin_plugin_paths = list(
+        _iter_plugin_metadata_files(BUILTIN_PLUGINS_DIR)
+    )
```

Comment:
if necessary, here you can assign the _builtin prop based off being listed from the builtin plugins dir